buildflags/BUILD.gn

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2018 GitHub, Inc.
+# Use of this source code is governed by the MIT license that can be
+# found in the LICENSE file.
+
+import("//build/buildflag_header.gni")
+import("//electron/buildflags/buildflags.gni")
+
+buildflag_header("buildflags") {
+  header = "buildflags.h"
+
+  flags = [
+    "ENABLE_DESKTOP_CAPTURER=$enable_desktop_capturer",
+    "ENABLE_RUN_AS_NODE=$enable_run_as_node",
+    "ENABLE_OSR=$enable_osr",
+    "ENABLE_VIEW_API=$enable_view_api",
+    "ENABLE_PEPPER_FLASH=$enable_pepper_flash",
+    "ENABLE_PDF_VIEWER=$enable_pdf_viewer",
+    "OVERRIDE_LOCATION_PROVIDER=$enable_fake_location_provider",
+  ]
+}
```

Comment:
Could this go in `build/` instead of in a new dir?

Reply:
I was planning to add it under `build/` but went with the directory structure upstream components use.